src/reducers/ProgressTrackerReducer.jsx

```diff
@@ -49,6 +49,20 @@ const ProgressTrackerReducer = (state = initialState, action) => {
       const ahseTotal = sum(ahseCourses);
       const sciTotal = sum(sciCourses);
 
+      // converts progress into a string that can be used by
+      // the progress bar in the ProgressTracker component
+      const mathSciPercNum = Math.round((mathTotal + sciTotal) * 3.33);
```

Comment:
It wasn't working in the component's curly braces and I couldn't figure out why. In the interest of time, I just left it similar to this.